### ISSUE TITLE
Accessibility Issues on Folder Content Management Screen

### ISSUE THREAD

**Issue by ma10** [plone/Products.CMFPlone#1194]
I have noticed the following accessibility issues on the folder content management screen (http://example.com/folder_contents)

These are on Plone 5.0 (Unified Installer) with Firefox and JAWS on Windows 8.1.  (The situation should be very similar with NVDA as well.)
## State of Buttons Are Not Properly Conveyed

When none of the files/folders are checked, buttons such as copy, cut, rename, delete, etc. are grayed out (visually).  but the disabled state of these buttons are not properly conveyed to screen readers.

To solve this, all buttons that are disabled should have "aria-disabled" attribute set to true, and whatever triggers buttons to be enabled, the aria-disabled attribute should be set to false.
## Modal (Popups) Dialogs Are Not Accessible

When one or more items are selected and buttons such as delete, rename, etc. are pressed, a modal dialog pops up (visually).  Because aria-hidden is set to true, these are not exposed to screen readers.

To properly address this situation, 
1. The element that contains the pop-up dialog should have `role="dialog"` set.
2. The keyboard focus should be moved to the dialog as it appears.
3. When the dialog disappears, the focus should be moved back to where it was right before the dialog appeared.
4. Either aria-label or aria-labelledby attribute should be used to convey what dialog is for to screen readers.

Please note that aria-hidden attribute is not to be used when there is other ways to hide the element from screen readers.  Right now, it seems that aria-hidden is used much more than it should be.

**Comment by terapyon** [comment]
I made PR `https://github.com/plone/mockup/pull/635`.
I think it fixed, but I can't detail check. 
@ma10 please help us.
